env/WCOSS2.env

```diff
@@ -90,13 +90,14 @@ elif [ $step = "eobs" ]; then
 
 elif [ $step = "eupd" ]; then
 
+    export OMP_PROC_BIND=true
     export OMP_PLACES=cores
     export OMP_STACKSIZE=2G
     export MPICH_COLL_OPT_OFF=1
     export FI_OFI_RXM_SAR_LIMIT=3145728
 
     export NTHREADS_ENKF=$nth_eupd
-    export APRUN_ENKF="$launcher -n ${npe_enkf:-${npe_eupd:-$PBS_NP}} -ppn $npe_node_eupd --cpu-bind depth --depth $NTHREADS_ENKF"
+    export APRUN_ENKF="$launcher -n ${npe_enkf:-${npe_eupd:-$PBS_NP}} -ppn $npe_node_eupd --cpu-bind core --depth $NTHREADS_ENKF"
```

Comment:
WCOSS2 testing indicates that `--cpu-bind core` yields unphysical `enkf.x` analysis increments.

Reply:
Test with `OMP_PROC_BIND=true` retained but replaced `core` with `depth` on `APRUN_ENKF` line so that `enkf.x` was run with `mpiexec -l -n 320 -ppn 10 --cpu-bind depth --depth 12`.  `enkf.x` wall time on Cactus was 1362.925156 seconds.  Physical analysis increments were generated.   Spot check of member analysis increments from this test show them to be identical with those from operational 20241002 12Z.

Repeat test with `OMP_PROC_BIND=true` removed.  `enkf.x` wall time dropped to 1250.748921 seconds.  Physical analysis increments produced.  Spot check of member increment files show them to be identical with operations.  Need more runs to confirm consistently faster wall time.